src/Select.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { theme } from 'styled-system'
+
+import { Box, Flex, Icon, Label } from '../src'
+
+const parseChevronIcon = isExpanded => {
+  const ChevronIcon = isExpanded
+    ? Icon.extend`
+        transform: rotate(180deg);
+      `
+    : Icon
+
+  return <ChevronIcon name="chevronDown" size={12} color="black" />
+}
+
+const SelectFlex = Flex.extend`
+  cursor: pointer;
+  height: 48px;
+  border-radius: ${theme('radius')};
+  border-width: 1px;
+  border-style: solid;
+  border-color: ${theme('colors.borderGray')};
+`
+
+const SelectBox = Box.extend`
+  cursor: pointer;
+`
+
+const SelectLabel = Label.extend`
+  cursor: pointer;
+`
+
+const Select = props => (
+  <SelectFlex justify="center" align="center">
+    <SelectBox pl={3}>
+      <Icon name="key" size={24} color="blue" />
```

Comment:
I'm guessing this icon should be configurable and optional?

Reply:
Totally nice suggestion. Updated to offer a `withIcon` boolean prop to allow client to opt in to use the icon, or opt out. Thanks~ ✌️